### DISCUSSION TITLE
An example where build and environment.yml are in 1 pixi.toml (emscripten-forge?)

### DISCUSSION THREAD

**Discussion by bugsbunny88** [prefix-dev/pixi#1057] [Q&A] [answered]
Out of curiosity can pixi be utilized to combine the 2 files into 1 . As a simple example ; https://github.com/jupyterlite/xeus-python-demo uses 2 files in their CI CD pipeline. I'm wondering if this pixi project is something where one can use say 1 toml file that I can utilize in a ci cd pipeline, and the same on my local windows machine . Or am I totally thinking about this wrong ?

**Comment by pavelzw** [comment]
Features and environments might solve your issues: https://pixi.sh/latest/configuration/#the-feature-and-environments-tables

**Reply by ruben-arts** [reply, marked as answer]
To add to this, you could checkout the blog that also has a similar example: https://prefix.dev/blog/introducing_multi_env_pixi

**Reply by bugsbunny88** [reply]
that blog post was extremely helpful ; many thanks to whoever put that together

**Comment by wolfv** [comment]
I think this would need some work in terms of cross-installing. There is an open PR
